packages/core/src/App/Containers/Modals/app-modals.jsx

```diff
@@ -125,6 +128,14 @@ const AppModals = observer(() => {
             ? mt5_login_list.find(login => login)?.white_label?.notification ?? true
             : false;
 
+    const { is_migrated } = useWalletMigration();
+
+    const { has_wallet } = useStoreWalletAccountsList();
+
+    const should_show_wallets_upgrade_completed_modal = localStorage.getItem(
+        'should_show_wallets_upgrade_completed_modal'
```

Comment:
would be really cool migrate away from local storage in favour of something which is not forever-persistent (so if something goes wrong, customers wont' be forced to clean their browser, as this is a pain even for technical ppl, and also I would personally really hate to be forced to loogin to all the pages again)

Reply:
I can try changing this in the PR for the post-migration UX because ig this was out of scope of WALL-2064: https://github.com/binary-com/deriv-app/pull/14472